model-optimizer/extensions/ops/scatter.py

```diff
@@ -94,6 +96,25 @@ class ScatterElementsUpdate(Scatter):
     op = op_type = 'ScatterElementsUpdate'
     version = 'opset3'
 
+    @staticmethod
+    def infer(node: Node):
+        Scatter.infer(node)
+
+        input_value = node.in_port(0).data.get_value()
+        indices_shape = node.in_port(1).data.get_shape()
+        indices_value = node.in_port(1).data.get_value()
+        updates_value = node.in_port(2).data.get_value()
+        axis = node.in_port(3).data.get_value()
```

Comment:
Please add checks that inputs are of the same rank. 
And one more question: can `indices` and `input` have different shapes?

Reply:
According to spec: The rank of indices tensor is equal to the rank of data tensor.

Reply:
I understand that the ranks of indices and data must be equal and this is reason why I asked to add assert statement to ensure that this is true. 
You can take as example here:
https://github.com/openvinotoolkit/openvino/blob/master/model-optimizer/extensions/ops/gatherelements.py#L48